srcpkgs/python3-paho_mqtt/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-paho_mqtt'
+pkgname=python3-paho_mqtt
+version=2.0.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="hatchling"
+depends="python3"
+short_desc="Eclipse Paho MQTT python client"
+maintainer="Andrew J. Hesford <ajh@sideband.org>"
+license="custom:EDL-1.0, custom:EPL-2.0"
```

Comment:
EPL-2.0 is [not custom](https://spdx.org/licenses/EPL-2.0.html)

Reply:
Thanks, I had misread "EPL" as "EDL".